src/webhook_launcher/processes/VCSCOMMIT_QUEUE

```diff
@@ -16,7 +16,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
-Ruote.process_definition 'vcsqueue' do
+Ruote.process_definition 'wh_handle_event' do
```

Comment:
I initially agreed and then thought that on the other hand having wh_handle_event in the Boss viewer etc correctly brings to mind handle_webhook and Payload.handle()
So given they are all working together I prefer to see the pdef, participant and code use the word "handle"... fair?